wremnants/syst_tools.py

```diff
@@ -318,6 +320,16 @@ def add_pdf_hists(results, df, dataset, axes, cols, pdfs, base_name="nominal", a
         else:
             pdfHist = df.HistoBoost(pdfHistName, axes, [*cols, tensorName], tensor_axes=[pdf_ax], storage=hist.storage.Double())
             alphaSHist = df.HistoBoost(alphaSHistName, axes, [*cols, tensorASName], tensor_axes=[as_ax], storage=hist.storage.Double())
+
+            if propagateToHelicity:
+                pdfhelper = ROOT.wrem.makeHelicityMomentPdfTensor[npdf]()
```

Comment:
Where is this defined? Is there a file missing in the commit?

Reply:
right... just added